app/assets/javascripts/forever_style_guide/list-toggle.js

```diff
@@ -1,18 +1,29 @@
-$(function () {
-  $(".list-group-toggle-trigger").click(function (e) {
-    e.preventDefault();
-    $(this).next('.list-group-item-child').toggleClass("is_open");
-    $("i", this).toggleClass("fa-plus fa-minus");
-  })
-});
+var setup_toggle_list = function(){
+    $(".list-group-toggler").click(function (e) {
```

Comment:
I'd guess no one shared this w/ you yet, but we prefer two space indents pretty much universally.
http://wiki.int.vpc/code-style-guide#JavaScript